.github/workflows/scanbuild_static-analysis.yml

```diff
@@ -145,8 +145,6 @@ jobs:
       run: git clean -xdf
     - name: Configuring Racket CS
       working-directory: ./racket/src
-      env:
```

Comment:
@pmatos 

There is no `matrix.cc` IIUC. Can this be safely removed?

Reply:
humm, when we stopped compiling with multiple compilers, that was left behind. Yes, pls remove.